pkgs/development/libraries/libsystemtap/default.nix

```diff
@@ -0,0 +1,27 @@
+{stdenv, fetchgit, gettext, python, elfutils}:
+
+stdenv.mkDerivation {
+  name = "libsystemtap";
```

Comment:
`pname` and `version` please.

Reply:
Couldn't find a version on the project, so calling it `1.0.0.`

Reply:
**If** there is no stable release the convention is to set the version to `unstable-YYYY-MM-DD` where the date is the date of the commit you're referencing.

Reply:
Good to know! It is actually tagged as `release-3.2` in the git repo, I was just being lazy.

Reply:
Updated version.